cmd/zed/lake/query/command.go

```diff
@@ -37,50 +34,41 @@ type Command struct {
 	lake        *zedlake.Command
 	stats       bool
 	stopErr     bool
-	at          string
 	includes    query.Includes
 	outputFlags outputflags.Flags
 	procFlags   procflags.Flags
-	searchFlags searchflags.Flags
 }
 
 func New(parent charm.Command, f *flag.FlagSet) (charm.Command, error) {
 	c := &Command{lake: parent.(*zedlake.Command)}
-	f.StringVar(&c.at, "at", "", "commit tag or journal ID for time travel")
```

Comment:
This feels like a useful option no?

Reply:
It's in the `from` statement now just like the pool name and the search key range, e.g., https://github.com/brimdata/zed/blob/25bfcdc807aca58a4747a9c8709f58d746bfb7cc/lake/ztests/time-travel.yaml#L9